### ISSUE TITLE
failure building with btrfs-progs 6.1

### ISSUE THREAD

**Issue by aminvakil** [goodwithtech/dockle#211]
**Description**
I maintain `dockle` package on AUR (https://github.com/goodwithtech/dockle/commit/15cfad27dfcff96e3209dc7495bb7f98b7baebf6) and testing that it builds and packages correctly every night on a clean Arch Linux instance.

Since yesterday night after `btrfs-progs` 6.1 has been released (6.0.2 was fine) this error started to happen:
```
/home/devel/go/pkg/mod/github.com/containers/storage@v1.38.2/drivers/btrfs/btrfs.go:385:11: args.lim.max_referenced undefined (type _Ctype_struct_btrfs_qgroup_limit has no field or method max_referenced)
```
I have also found this https://github.com/moby/moby/issues/44698 which may be relevant.

**What did you expect to happen?**
https://github.com/aminvakil/aur/actions/runs/3778493275/jobs/6423010765#step:11:89 (Last successful run)

**What happened instead?**
https://github.com/aminvakil/aur/actions/runs/3784578791/jobs/6433988661#step:11:173

**Output of run with `-debug`:**
Irrelevant.

**Output of `dockle -v`:**
Irrelevant.

**Additional details (base image name, container registry info...):**
Build section is like this, full `PKGBUILD` is here: https://github.com/aminvakil/aur/blob/master/dockle/PKGBUILD
```
  export CGO_CPPFLAGS="${CPPFLAGS}"
  export CGO_CFLAGS="${CFLAGS}"
  export CGO_CXXFLAGS="${CXXFLAGS}"
  export CGO_LDFLAGS="${LDFLAGS}"
  export GOFLAGS="-buildmode=pie -trimpath -ldflags=-linkmode=external -mod=readonly -modcacherw"
  go build -o build ./cmd/dockle
```

**Comment by tomoyamachi** [comment]
Thank you for reporting!
Many libraries used by Dockle depend on moby/moby. [This line(docker/docker)](https://github.com/goodwithtech/dockle/blob/9b577345701e3791e9a3ffa3699d18315e3e30d4/go.mod#L36) in the go.mod file is an alias for moby/moby.
We need to wait for the related libraries to upgrade with [the moby's fix](https://github.com/moby/moby/pull/44707). Then we will fix the building bug.

**Comment by aminvakil** [comment]
https://github.com/moby/moby/releases/tag/v20.10.23 has been released last week.

**Comment by tomoyamachi** [comment]
@aminvakil Thank you for your notification.
 I've fixed it on v0.4.11.
